src/stitching/introspectSchema.ts

```diff
@@ -1,10 +1,11 @@
 import { GraphQLSchema, DocumentNode } from 'graphql';
-import { introspectionQuery, buildClientSchema, parse } from 'graphql';
+import { buildClientSchema, parse } from 'graphql';
+import { getIntrospectionQuery } from 'graphql/utilities';
```

Comment:
No need to change this before merging this PR, but we could make these imports more specific:
```ts
import { parse } from 'graphql/language/parser';
import { buildClientSchema } from 'graphql/utilities/buildClientSchema';
import { getIntrospectionQuery } from 'graphql/utilities/introspectionQuery';
```
That said, what you've done is already an improvement over importing `from 'graphql'`.